cmd/gardener-extension-runtime-gvisor/app/app.go

```diff
@@ -106,6 +108,16 @@ func NewControllerManagerCommand(ctx context.Context) *cobra.Command {
 		},
 	}
 
+	// Add version subcommand
+	cmd.AddCommand(
+		&cobra.Command{
+			Use:   "version",
+			Short: "Print the program version number",
```

Comment:
```suggestion
			Short: fmt.Sprintf("Print the %s version", gvisor.Name),
```

Reply:
No longer applicable after switching from custom code to k8s' verflag.